### PR TITLE
add test description for AWS SecurityHub Scan

### PR DESCRIPTION
**Pull request by manuel-sommer**

### CODE REVIEW

**Comment by mtesauro** [comment]
@manuel-sommer You might want to look at @cneill 's points in #9912 - not the same thing as you're doing here but also about adding items to the description of AWS Sec Hub

**Comment by manuel-sommer** [comment]
I looked at the PR @mtesauro and reviewed it.

File: dojo/tools/awssecurityhub/parser.py [hunk 1]
```diff
@@ -2,9 +2,11 @@
 from dojo.tools.awssecurityhub.inspector import Inspector
 from dojo.tools.awssecurityhub.guardduty import GuardDuty
 from dojo.tools.awssecurityhub.compliance import Compliance
+from dojo.tools.parser_test import ParserTest
 
 
 class AwsSecurityHubParser:
+    ID = "AWS SecurityHub"
```
Comment (cneill) [inline dojo/tools/awssecurityhub/parser.py:9]:
```suggestion
    ID = "AWS Security Hub"
```

**Review by cneill — CHANGES_REQUESTED** [review]
Minor change on the Test name but otherwise looks good!

**Comment by manuel-sommer** [comment]
Done @cneill

**Review by cneill — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by blakeaowens — APPROVED** [review]

**Review by Maffooch — APPROVED** [review]

**Pull request merged**